guidelines/sc/22/consistent-help.html

```diff
@@ -15,7 +15,7 @@ <h4>Consistent Help</h4>
     </ul>
 
   <p class="note">Access to help mechanisms may be provided directly on the page, or may be provided via a direct link to a different page containing the information.</p>
-  <p class="note">For this Success Criterion, the same relative order can be thought of as how the content is ordered when the page is serialized. The visual position of a help mechanism is likely to be consistent across pages for the same page variation (e.g., CSS break-point). The user can cause a change, for example zoom or change orientation, which may trigger a different page variation.</p>
+  <p class="note">For this Success Criterion, the same relative order can be thought of as how the content is ordered when the page is serialized. The visual position of a help mechanism is likely to be consistent across pages for the same page variation (e.g., CSS break-point). The user can initiate a change, such as changing the page's zoom or orientation, which may trigger a different page variation. This criterion is concerned with relative order across pages at the same zoom level and orientation.</p>
```

Comment:
liking the now wording much better. less confusing. but in the second sentence, perhaps go back to the page variation idea, rather than zoom/orientation (which are only two examples of user-triggered variation).

`This criterion is concerned with relative order across pages displayed in the same page variation (e.g. same zoom level and orientation).`